### PR TITLE
FeedTableViewCell animation continues if song is playing and feed is …

### LINKED ISSUE #45: Player stops animating after pull to refresh
After pull to refresh, the tableview reloads and any cell that is playing/animating stops animating but keeps on playing.

### PR DESCRIPTION
**Pull request by ChenJesse**
…reloaded

Fixes #45

### CODE REVIEW

**Comment by anniexcheng** [comment]
Code-wise, looks good! @keivanshahida @loganallen Is the fix visible on your ends?

File: Tempo/Controllers/FeedViewController.swift [hunk 1]
```diff
@@ -159,6 +160,25 @@ class FeedViewController: PlayerTableViewController, SongSearchDelegate, PostVie
 		refreshFeedWithDelay(1.0, timeout: 5.0)
 	}
 	
+	func continueAnimatingAfterRefresh() {
+		//animate currently playing song
+		if let playerCellPost = (self.navigationController as! PlayerNavigationController).playerCell.post {
+			let playingSong = playerCellPost.song
+			for row in 0 ..< posts.count {
+				let cell = tableView.cellForRowAtIndexPath(NSIndexPath(forRow: row, inSection: 0)) as! FeedTableViewCell
+				if let cellPost = cell.postView.post {
```
Comment (dennisfedorko) [inline Tempo/Controllers/FeedViewController.swift]: should squash the indentation level

if let cellPost = cell.postView.post where cellPost.song.spotifyID == playingSong.spotifyID {
    //animate the appropriate post view
}

**Comment by ChenJesse** [comment]
Ok I think Keivan and Logan should get their things in first because they are larger changes, and then I'll make Dennis' change

File: Tempo/Controllers/PlayerTableViewController.swift [hunk 2]
```diff
@@ -30,24 +31,24 @@ class PlayerTableViewController: UIViewController, UITableViewDelegate, UITableV
 			if searchController.active {
 				array = filteredPosts
 			}
-            if let row = currentlyPlayingIndexPath?.row where currentlyPlayingPost?.isEqual(array[row]) ?? false {
-                currentlyPlayingPost?.player.togglePlaying()
+			if let row = currentlyPlayingIndexPath?.row where (array[row].isEqual(playerNav.playerCell.post) ?? false) {
```
Comment (lucasderraugh) [inline Tempo/Controllers/PlayerTableViewController.swift]: Why is `isEqual` needed here? Does `==` not suffice for some reason?

Comment (ChenJesse) [reply Tempo/Controllers/PlayerTableViewController.swift]: Not necessary, I'll change it then

**Comment by ChenJesse** [comment]
Ok, fixed a few more bugs, I think it should be ready to merge

**Comment by anniexcheng** [comment]
yup looks good to merge!

**Pull request merged**